### PR TITLE
Added webpack5 support

### PR DESCRIPTION
**Pull request by Demonkratiy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,19 +54,19 @@
     "lodash.findindex": "4.6.0",
     "lodash.isequal": "4.5.0",
     "lodash.template": "4.5.0",
-    "mini-css-extract-plugin": "^0.9.0",
+    "mini-css-extract-plugin": "^1.4.0",
     "multi-loader": "0.1.0",
     "powerbi-visuals-webpack-plugin": "^2.2.5",
```
Comment (uve) [inline package.json:59]: Use the latest powerbi-visuals-webpack-plugin v2.3.0

**Review by uve — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Demonkratiy) [reply package.json]: implemented latest version 2.2.6

File: package.json [hunk 2]
```diff
@@ -54,19 +54,19 @@
     "lodash.findindex": "4.6.0",
     "lodash.isequal": "4.5.0",
     "lodash.template": "4.5.0",
```
[the remark this answers is not in the document]
Comment (Demonkratiy) [reply package.json]: removed lodash: template/difference/findindex

File: config.json [hunk 3]
```diff
@@ -1,32 +1 @@
...
-        "certificate": "certs/PowerBICustomVisualTest_public.crt",
-        "pfx": "certs/PowerBICustomVisualTest_public.pfx",
-        "port": "8080"
-    },
-    "visualTemplates": {
-        "circlecard": "http://github.com/microsoft/powerbi-visuals-circlecard-react/archive/master.zip"
-    }
-}
\ No newline at end of file
+{"templates":{"visuals":"templates/visuals","pbiviz":"templates/pbiviz.json.template","plugin":"templates/plugin.ts.template","package":"templates/package.json.template"},"generate":{"apiVersion":"2.6.0"},"build":{"precompileFolder":".tmp/precompile","dropFolder":".tmp/drop","js":"tmp.visual.js","css":"visual.css","stats":"../../webpack.statistics.html"},"package":{"dropFolder":"dist"},"server":{"root":"webRoot","assetsRoute":"/assets","privateKey":"certs/PowerBICustomVisualTest_private.key","certificate":"certs/PowerBICustomVisualTest_public.crt","pfx":"certs/PowerBICustomVisualTest_public.pfx","port":"8080","passphrase":"2026086985663691"},"visualTemplates":{"circlecard":"http://github.com/microsoft/powerbi-visuals-circlecard-react/archive/master.zip"}}
```
[the remark this answers is not in the document]
Comment (Demonkratiy) [reply config.json]: Decided to implement API 3.6.0 in next release

**Pull request merged**